src/sample/stuDashController.java

```diff
@@ -72,7 +78,7 @@ public void editStuInfo() {
     minor.setDisable(false);
   }
 
-  public void saveInfo() {
+  public void saveInfo() throws IOException{
```

Comment:
Why throws IOException

Reply:
it involves files, so i put throws IOException, but i also think i might be salah concept